### PR TITLE
documentation to static-pods for naming of pods

### PR DESCRIPTION
**Pull request by Xorima**
This will add documentation around how a static pod will be named

### CODE REVIEW

**Comment by ydFu** [comment]
@Xorima 

Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Review by Aut0R3V — APPROVED** [review]

**Comment by bradtopol** [comment]
@Xorima Thank you for your contribution. Please sign the CLA so we can accept your PR for review.
/assign

**Comment by Xorima** [comment]
@bradtopol cla is now signed, sorry for the delay had some internal organisation questions come up.

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The kubelet automatically tries to create a {{< glossary_tooltip text="mirror Po
 on the Kubernetes API server for each static Pod.
 This means that the Pods running on a node are visible on the API server,
 but cannot be controlled from there.
+The Pods' name will suffixed with the node hostname with a leading hyphen
```
Comment (onlydole) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:25]:
```suggestion
The Pod names will suffixed with the node hostname with a leading hyphen
```

Howdy, @Xorima! I'd recommend changing the possessive such that it reads like this suggestion, and provides a bit more clarity.

**Review by onlydole — APPROVED** [review]
LGTM! Thank you, @Xorima 🎉

**Pull request merged**